src/components/VoteScreen.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import Title from './Title';
+import VoteRow from './VoteRow';
+import axios from 'axios';
```

Comment:
import문 이렇게 분리해주시면 나중에 보기 좋을 것 같아요~

```suggestion
import React, { useState, useEffect } from 'react';
import styled from 'styled-components';
import axios from 'axios';

import Title from './Title';
import VoteRow from './VoteRow';

```